From 89dd3063259aea6081d83ce613691525bd5c3090 Mon Sep 17 00:00:00 2001
From: Nathan Chancellor <nathan@kernel.org>
Date: Tue, 30 Nov 2021 15:43:45 -0700
Subject: [PATCH 12/18] HACK: 9p: Fix unused variable warning in
 v9fs_vfs_readahead()

clang warns:

fs/9p/vfs_addr.c:140:16: error: unused variable 'inode' [-Werror,-Wunused-variable]
        struct inode *inode = folio_inode(folio);
                      ^
1 error generated.

Link: https://lore.kernel.org/r/20211130152753.6899aa0c@canb.auug.org.au/
Signed-off-by: Nathan Chancellor <nathan@kernel.org>
---
 fs/9p/vfs_addr.c | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/fs/9p/vfs_addr.c b/fs/9p/vfs_addr.c
index 1279970e9157..3281ce6af61f 100644
--- a/fs/9p/vfs_addr.c
+++ b/fs/9p/vfs_addr.c
@@ -137,7 +137,6 @@ static void v9fs_vfs_readahead(struct readahead_control *ractl)
 static int v9fs_release_page(struct page *page, gfp_t gfp)
 {
 	struct folio *folio = page_folio(page);
-	struct inode *inode = folio_inode(folio);
 
 	if (folio_test_private(folio))
 		return 0;
@@ -147,7 +146,7 @@ static int v9fs_release_page(struct page *page, gfp_t gfp)
 			return 0;
 		folio_wait_fscache(folio);
 	}
-	fscache_note_page_release(v9fs_inode_cookie(V9FS_I(inode)));
+	fscache_note_page_release(v9fs_inode_cookie(V9FS_I(folio_inode(folio))));
 #endif
 	return 1;
 }
-- 
2.34.1

